src/wp-includes/default-filters.php

```diff
@@ -757,8 +757,9 @@
 add_filter( 'rest_pre_insert_wp_template', 'inject_ignored_hooked_blocks_metadata_attributes' );
 add_filter( 'rest_pre_insert_wp_template_part', 'inject_ignored_hooked_blocks_metadata_attributes' );
 
-// Update ignoredHookedBlocks postmeta for wp_navigation post type.
+// Update ignoredHookedBlocks postmeta for wp_navigation and wp_template_part post types.
 add_filter( 'rest_pre_insert_wp_navigation', 'update_ignored_hooked_blocks_postmeta' );
+add_filter( 'rest_pre_insert_wp_template_part', 'update_ignored_hooked_blocks_postmeta' );
```

Comment:
Looking at L758, I noticed that we're already attaching another filter (`inject_ignored_hooked_blocks_metadata_attributes`) to same hook 😅 

We probably shouldn't do both but decide which one to go with, and modify such that it covers both "normal" hooked block injection, and top-level first_child/last_child injection 🤔

Reply:
Off the top of my head, template parts have more in common with templates (as they both require the `stdClass` ➡️  `WP_Post` ➡️ `WP_Block_Template` conversion), so maybe we should go with that (and remove the `update_ignored_hooked_blocks_postmeta` filter after all 🙈 

My hope is that we'll be able to find some helper function(s) that is used one level "down" (inside of `update_ignored_hooked_blocks_postmeta`) for setting the postmeta that we'll be able to reuse.

Reply:
I've pushed a couple of commits to make that change.